daemon/src/document.rs

```diff
@@ -24,12 +24,12 @@ pub struct Document {
 }
 
 impl Document {
-    pub fn new(init: bool) -> Self {
+    pub fn new(init_structure: bool) -> Self {
```

Comment:
`init` is overloaded, so I'm trying to specify what exactly.

reasoning: I thought it's misleading to call this parameter `init`, as the flag.
Initializing a new document (this whole constructor) != initializing the automerge objects. The latter is an optional sub operation of the former.

Reply:
I like it!